src/test/java/MainTest.java

```diff
@@ -9,4 +12,43 @@ public void testSth() {
         assertTrue(true);
     }
 
+
+    /**
+     * Method check setting path
+     * */
+    @Test
+    public void testFileManipulatorSetter() {
+        WordFinder fm = WordFinder.getInstance();
+
+        fm.setPath("data/temporary.txt");
+
+        assertTrue(fm.getPath().equals("data/temporary.txt"));
+    }
+
+
+    /**
+     * Method check sizeof List
+     * */
+    @Test
+    public void testListInWordFinder() {
+        WordFinder fm = WordFinder.getInstance();
+
+        fm.prepareKeywords();
+
+        assertTrue(fm.getSize() == 12);
+    }
+
+    /**
+     * Method check reading file
+     * */
+    @Test
+    public void testReadingInWordFinder() {
+        WordFinder fm = WordFinder.getInstance();
+        fm.setPath("data/test.txt");
+        try {
+            assertTrue(fm.readFile() == 2);
+        } catch (IOException e) {
+            System.err.println(e.getMessage());
+        }
```

Comment:
W phpUnit da się wymusić "chęć sprawdzenia" czy został wyrzucony dany wyjątek. Patrzyłem jak to jest w jUnit - podobno da się przez adnotacje, ale mi nie zadziałało, więc możemy zostawić tak jak jest.

Zamień tylko assertTrue na assertEquals();